wiki/BanchoBot/en.md

```diff
@@ -1,21 +1,15 @@
-<!-- BanchoBot Information -->
 # BanchoBot
 
-![BanchoBot's player card](BanchoBot.jpg "BanchoBot's player card")
+![BanchoBot's usercard](BanchoBot.jpg "BanchoBot's usercard")
 
-BanchoBot is a bot (i.e. a set of commands with automated responses) programmed by **Echo49** that assists people in chat by displaying certain information and announcing game-related messages.
-It is the host of the [bancho irc](/wiki/Internet_Relay_Chat) and has a user profile with the ID _[#3](https://osu.ppy.sh/u/3)_ (next to peppy who is [#2](https://osu.ppy.sh/u/2)). *It also has its own [twitter profile](https://twitter.com/banchoboat)!*
+BanchoBot is a bot (i.e. a set of commands with automated responses) programmed by [Echo49](/users/431) that assists people in chat by displaying certain information and announcing game-related messages. BanchoBot is the host of the [bancho IRC](/wiki/Internet_Relay_Chat) and has a [user profile](/users/3). BanchoBot also has its own [twitter profile](https://twitter.com/banchoboat)!
 
-
-<!-- BanchoBot (server) commands start with an exclamation mark! `!`-->
 ## BanchoBot Commands
 
-To use BanchoBot commands, you must use the `!` prefix everytime. **Normal users can only use these commands in multiplayer channels and in BanchoBot PM.** If a user send a command in public chat, normal users won't see it and they get it as a private message.
-Users can also use `/bb <command>` client command to automatically open a BanchoBot tab and sending the command immediately.
+*For a list of client commands, see: [Chat Console](/wiki/Chat_Console#commands-list)*
 
-*For a list of slash (client) commands, visit the [Chat Console](/wiki/Chat_Console#commands-list) article.*
+To use BanchoBot commands, you must use the `!` prefix everytime. **Normal users can only use these commands in multiplayer channels and in BanchoBot PM.** If a user send a command in public chat, normal users won't see it and they get it as a private message. Users can also use `/bb <command>` client command to automatically open a BanchoBot tab and sending the command immediately.
 
-<!-- Commands List -->
 | Command   | Argument(s)            | Description | Example Input | Example Response |
```

Comment:
Fix the table format (i.e. spacing, delimiter, etc.).

Reply:
There is HTML inside the table, I'm not sure how to approach this.

Reply:
There is HTML inside the table. I'm not sure how to approach this, since it should be reformatted altogether. So I'm going to leave it as is for someone else who can think of a better way to format it.

Reply:
Hmm, never mind, will reformat with headings. The way I was trying previously looked weird so I trashed it.